apps/web/frontend/generated/graphql.ts

```diff
@@ -365,6 +370,7 @@ export type User = {
   __typename: 'User'
   /** Capacity of the user in the team */
   availableMinutesPerWeek?: Maybe<Scalars['Int']>
+  durationWorkedOnProject: Scalars['Int']
```

Comment:
? name not clear to me. is it minutesWorked ?

Reply:
So, right now we are using the word `duration` for minutes worked on something. For example here: https://github.com/progwise/timebook/blob/main/packages/backend/src/graphql/workHour/workHourInput.ts#L10

Reply:
So I looked it up:
We used the word `duration` on GraphQL fields five times.
We used the word `minutes` on GraphQL fields two times, but in a different context (`availableMinutesPerWeek`).

I don't see the benefit of using a different term here?